dev/integration_tests/semantics/test_driver/android_testing.dart

```diff
@@ -0,0 +1,512 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+
+import 'package:test/test.dart';
+
+
+/// ClassName constants used by Flutter semantics.
+class AndroidClassName {
+  /// A checkbox widget.
+  static const String checkBox = 'android.widget.CheckBox';
+
+  /// The default className if none is provided by flutter.
+  static const String view = 'android.view.View';
+
+  /// A radio widget.
+  static const String radio = 'android.widget.RadioButton';
+
+  /// An editable text widget.
+  static const String editText = 'android.widget.EditText';
+
+  /// A read-only text widget.
+  static const String textView = 'android.widget.TextView';
+}
+
+/// Action constants Taken from [AccessibilityAction].
```

Comment:
This is Android's `android.view.accessibility.AccessibilityNodeInfo.AccessibilityAction`? Maybe make that clearer? Also, I am not sure if dart doc can link to that, so maybe use enclose it in `` instead of []?